### ISSUE TITLE
ReferenceError: AbortController is not defined

### ISSUE THREAD

**Issue by Vinnl** [nolanlawson/fuite#4]
Fuite sounds great! Unfortunately I can't get it to run - is there a particular Node version required? Here's the error output:

```
$ npx fuite --debug https://pinafore.social
ReferenceError: AbortController is not defined
    at main (file:///home/vincent/.npm/_npx/4f0bae50eb4a6363/node_modules/fuite/src/cli.js:31:3)
    at file:///home/vincent/.npm/_npx/4f0bae50eb4a6363/node_modules/fuite/src/cli.js:114:1
    at ModuleJob.run (internal/modules/esm/module_job.js:170:25)
    at async Loader.import (internal/modules/esm/loader.js:178:24)
```

This is using 14.17.5 and npm 7.21.0.

**Comment by nolanlawson** [comment]
Yup, I just noticed this too. I need to add a polyfill for Node 14.

**Comment by nolanlawson** [comment]
Fixed in v1.0.1
